pkg/block/mem/adapter_test.go

```diff
@@ -1,14 +1,7 @@
 package mem_test
 
-import (
-	"testing"
-
-	"github.com/treeverse/lakefs/pkg/block"
-	"github.com/treeverse/lakefs/pkg/block/mem"
-	adapterTest "github.com/treeverse/lakefs/pkg/block/test"
-)
-
-func TestMemAdapter(t *testing.T) {
-	var params []func(*mem.Adapter)
-	adapterTest.TestAdapter(t, block.BlockstoreTypeMem, params)
-}
+// TODO (niro): Need to enable
```

Comment:
A marker so you won't forget

Reply:
https://github.com/treeverse/lakeFS/issues/5325